redbot/core/utils/chat_formatting.py

```diff
@@ -120,7 +122,7 @@ def inline(text: str) -> str:
         return "`{}`".format(text)
 
 
-def italics(text: str) -> str:
+def italics(text: str, escape_formatting: bool = True) -> str:
     """Get the given text in italics.
 
     Note: This escapes text prior to italicising
```

Comment:
This needs an update, maybe:
```suggestion
    Note: By default, this function will escape ``text`` prior to italicising.
```